### ISSUE TITLE
Trusted domains configuration bugs

### ISSUE THREAD

**Issue by szekelyisz** [nextcloud/nextcloudpi#1408]
Using NextcloudPi as a container. When you don't fill all 3 options for trusted domains, they are not applied correctly. Filling only the first two entries and clicking Apply results int he following messages:

```
[ nc-trusted-domains ] (Sun Dec 26 19:03:12 UTC 2021)
Cannot load Zend OPcache - it was already loaded
System config value trusted_domains => 22 set to string deley-nextcloud.local
Cannot load Zend OPcache - it was already loaded
System config value trusted_domains => 22 set to string nextcloud.deley.hu
```

And a red dot besides the Apply button. Filling the 3rd entry with 127.0.0.1 seems to change settings correctly. The dot turns green.

After you set the 3rd name and apply, it still doesn't work, I get the "Untrusted domain" error on the nextcloud interface. Restarting the container doesn't help. `config.php` contains:

```
  'trusted_domains' => 
  array (
    0 => 'localhost',
    22 => '127.0.0.1',
    11 => <my external ip address>,
    1 => '172.17.0.2',
    3 => 'f53a549f79ac',
  ),
```

Which is not the 3 domains that I configured.

**System information**

<details>
<summary>NextCloudPi diagnostics</summary>

```

Cannot load Zend OPcache - it was already loaded
Cannot load Zend OPcache - it was already loaded
NextCloudPi version  v1.40.4
NextCloudPi image    NextCloudPi_10-07-21
distribution         Debian GNU/Linux 10 \n \l
automount            no
USB devices          none
datadir              /data/nextcloud/data
data in SD           yes
data filesystem      overlayfs
data disk usage      3.8G/15G
rootfs usage         3.8G/15G
swapfile             /var/swap
dbdir                /data/database
Nextcloud check      ok
Nextcloud version    21.0.4.1
HTTPD service        up
PHP service          up
MariaDB service      up
Redis service        up
HPB service          down
Postfix service      up
internet check       ok
port check 80        closed
port check 443       closed
IP                   ***REMOVED SENSITIVE VALUE***
gateway              ***REMOVED SENSITIVE VALUE***
interface            eth0
certificates         ***REMOVED SENSITIVE VALUE***
NAT loopback         no
uptime               2:20

```
</details>

<details>
<summary>Nextcloud configuration</summary>

```

Cannot load Zend OPcache - it was already loaded
{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": {
            "0": "localhost",
            "22": "127.0.0.1",
            "11": "5.38.215.73",
            "1": "172.17.0.2",
            "3": "f53a549f79ac"
        },
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "21.0.4.1",
        "overwrite.cli.url": "https:\/\/f53a549f79ac\/",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 0,
            "password": "***REMOVED SENSITIVE VALUE***"
        },
        "tempdirectory": "\/var\/www\/nextcloud\/data\/tmp",
        "mail_smtpmode": "sendmail",
        "mail_smtpauthtype": "LOGIN",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "preview_max_x": "2048",
        "preview_max_y": "2048",
        "jpeg_quality": "60",
        "overwriteprotocol": "https",
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "htaccess.RewriteBase": "\/"
    }
}

```
</details>

<details>
<summary>HTTPd logs</summary>

```

[Sun Dec 26 19:03:06.819367 2021] [authz_host:error] [pid 113:tid 2765493280] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /index.php failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:07.061845 2021] [proxy_fcgi:error] [pid 113:tid 2765493280] [client 192.168.1.161:46754] AH01071: Got error 'PHP message: PHP Warning:  file_get_contents(/usr/local/etc/ncp-config.d/nc-init.cfg): failed to open stream: No such file or directory in /var/www/ncp-web/elements.php on line 181PHP message: PHP Warning:  sizeof(): Parameter must be an array or an object that implements Countable in /var/www/ncp-web/elements.php on line 188PHP message: PHP Warning:  file_get_contents(/usr/local/etc/ncp-config.d/nc-init.cfg): failed to open stream: No such file or directory in /var/www/ncp-web/elements.php on line 136PHP message: PHP Warning:  Invalid argument supplied for foreach() in /var/www/ncp-web/elements.php on line 21'
[Sun Dec 26 19:03:07.068549 2021] [authz_host:error] [pid 113:tid 2807456800] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /js/ncp.js failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:07.069676 2021] [authz_host:error] [pid 113:tid 2765493280] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /js/minified.js failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:07.070509 2021] [authz_host:error] [pid 113:tid 2773885984] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /img/ncp-logo.svg failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:07.071301 2021] [authz_host:error] [pid 113:tid 2782278688] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /img/loading-small.gif failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:07.072900 2021] [authz_host:error] [pid 113:tid 2757100576] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /css/ncp.css failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:07.311627 2021] [authz_host:error] [pid 113:tid 2765493280] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /img/defaults.svg failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:07.311627 2021] [authz_host:error] [pid 113:tid 2757100576] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /img/toggle.svg failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:07.311628 2021] [authz_host:error] [pid 113:tid 2782278688] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /img/poweroff.svg failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:07.311895 2021] [authz_host:error] [pid 113:tid 2765493280] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /img/reboot.svg failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:07.340782 2021] [authz_host:error] [pid 113:tid 2782278688] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /ncp-launcher.php failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:07.340782 2021] [authz_host:error] [pid 113:tid 2765493280] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /ncp-output.php failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:10.202508 2021] [authz_host:error] [pid 113:tid 2757100576] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /ncp-launcher.php failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:11.561049 2021] [authz_host:error] [pid 113:tid 2782278688] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /ncp-launcher.php failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:13.644817 2021] [authz_host:error] [pid 113:tid 2782278688] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /ncp-launcher.php failed, reason: unable to get the remote host name
[Sun Dec 26 19:03:18.732776 2021] [proxy_fcgi:error] [pid 113:tid 2782278688] [client 192.168.1.161:46754] AH01071: Got error 'PHP message: PHP Warning:  file_get_contents(/usr/local/etc/ncp-config.d/nc-init.cfg): failed to open stream: No such file or directory in /var/www/ncp-web/elements.php on line 181'
[Sun Dec 26 19:05:17.880927 2021] [authz_host:error] [pid 113:tid 2782278688] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /ncp-launcher.php failed, reason: unable to get the remote host name
[Sun Dec 26 19:05:20.744531 2021] [authz_host:error] [pid 113:tid 2782278688] [client 192.168.1.161:46754] AH01753: access check of 'localhost' to /ncp-launcher.php failed, reason: unable to get the remote host name
[Sun Dec 26 19:05:25.736446 2021] [proxy_fcgi:error] [pid 113:tid 2782278688] [client 192.168.1.161:46754] AH01071: Got error 'PHP message: PHP Warning:  file_get_contents(/usr/local/etc/ncp-config.d/nc-init.cfg): failed to open stream: No such file or directory in /var/www/ncp-web/elements.php on line 181'

```
</details>

<details>
<summary>Database logs</summary>

```

2021-12-26 18:59:43 0 [Note] InnoDB: Uses event mutexes
2021-12-26 18:59:43 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
2021-12-26 18:59:43 0 [Note] InnoDB: Number of pools: 1
2021-12-26 18:59:43 0 [Note] InnoDB: Using generic crc32 instructions
2021-12-26 18:59:43 0 [Note] InnoDB: Initializing buffer pool, total size = 256M, instances = 1, chunk size = 128M
2021-12-26 18:59:43 0 [Note] InnoDB: Completed initialization of buffer pool
2021-12-26 18:59:43 0 [Note] InnoDB: If the mysqld execution user is authorized, page cleaner thread priority can be changed. See the man page of setpriority().
2021-12-26 18:59:43 0 [Note] InnoDB: 128 out of 128 rollback segments are active.
2021-12-26 18:59:43 0 [Note] InnoDB: Creating shared tablespace for temporary tables
2021-12-26 18:59:43 0 [Note] InnoDB: Setting file './ibtmp1' size to 12 MB. Physically writing the file full; Please wait ...
2021-12-26 18:59:43 0 [Note] InnoDB: File './ibtmp1' size is now 12 MB.
2021-12-26 18:59:43 0 [Note] InnoDB: 10.3.27 started; log sequence number 3183198; transaction id 2696
2021-12-26 18:59:43 0 [Note] InnoDB: Loading buffer pool(s) from /data/database/ib_buffer_pool
2021-12-26 18:59:43 0 [Note] Plugin 'FEEDBACK' is disabled.
2021-12-26 18:59:44 0 [Note] Server socket created on IP: '127.0.0.1'.
2021-12-26 18:59:44 0 [Note] Reading of all Master_info entries succeeded
2021-12-26 18:59:44 0 [Note] Added new Master_info '' to hash table
2021-12-26 18:59:44 0 [Note] mysqld: ready for connections.
Version: '10.3.27-MariaDB-0+deb10u1'  socket: '/run/mysqld/mysqld.sock'  port: 3306  Debian 10
2021-12-26 18:59:44 0 [Note] InnoDB: Buffer pool(s) load completed at 211226 18:59:44

```
</details>

<details>
<summary>Nextcloud logs</summary>

```

{"reqId":"pS9HjPq8VWaJyQsTPwfO","level":3,"time":"2021-10-07T20:25:01+00:00","remoteAddr":"","user":"--","app":"cron","method":"","url":"--","message":{"Exception":"Exception","Message":"Not installed","Code":0,"Trace":[{"file":"/data-ro/nextcloud/lib/base.php","line":643,"function":"checkInstalled","class":"OC","type":"::","args":[{"__class__":"OC\\SystemConfig"}]},{"file":"/data-ro/nextcloud/lib/base.php","line":1076,"function":"init","class":"OC","type":"::","args":[]},{"file":"/data-ro/nextcloud/cron.php","line":43,"args":["/data-ro/nextcloud/lib/base.php"],"function":"require_once"}],"File":"/data-ro/nextcloud/lib/base.php","Line":276,"CustomMessage":"--"},"userAgent":"--","version":""}
{"reqId":"hFGAKNvfQarjaxXVfGn3","level":3,"time":"2021-10-07T20:26:59+00:00","remoteAddr":"","user":"ncp","app":"no app in context","method":"","url":"--","message":{"Exception":"OCP\\AppFramework\\QueryException","Message":"Could not resolve trashManager! Class trashManager does not exist","Code":0,"Trace":[{"file":"/data-ro/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":126,"function":"resolve","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["trashManager"]},{"file":"/data-ro/nextcloud/lib/private/ServerContainer.php","line":162,"function":"query","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["trashManager",true]},{"file":"/data-ro/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":434,"function":"query","class":"OC\\ServerContainer","type":"->","args":["trashManager",true]},{"file":"/data-ro/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":56,"function":"query","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->","args":["trashManager"]},{"file":"/data-ro/nextcloud/lib/private/AppFramework/Bootstrap/FunctionInjector.php","line":58,"function":"get","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["trashManager"]},{"function":"OC\\AppFramework\\Bootstrap\\{closure}","class":"OC\\AppFramework\\Bootstrap\\FunctionInjector","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/data-ro/nextcloud/lib/private/AppFramework/Bootstrap/FunctionInjector.php","line":68,"function":"array_map","args":[{"__class__":"Closure"},[{"name":"serverContainer","__class__":"ReflectionParameter"},{"name":"logger","__class__":"ReflectionParameter"},{"name":"appManager","__class__":"ReflectionParameter"},"*** sensitive parameter replaced ***"]]},{"file":"/data-ro/nextcloud/lib/private/AppFramework/Bootstrap/BootContext.php","line":52,"function":"injectFn","class":"OC\\AppFramework\\Bootstrap\\FunctionInjector","type":"->","args":[[{"__class__":"OCA\\Files_Trashbin\\AppInfo\\Application"},"registerTrashBackends"]]},{"file":"/data-ro/nextcloud/apps/files_trashbin/lib/AppInfo/Application.php","line":57,"function":"injectFn","class":"OC\\AppFramework\\Bootstrap\\BootContext","type":"->","args":[[{"__class__":"OCA\\Files_Trashbin\\AppInfo\\Application"},"registerTrashBackends"]]},{"file":"/data-ro/nextcloud/lib/private/AppFramework/Bootstrap/Coordinator.php","line":178,"function":"boot","class":"OCA\\Files_Trashbin\\AppInfo\\Application","type":"->","args":[{"__class__":"OC\\AppFramework\\Bootstrap\\BootContext"}]},{"file":"/data-ro/nextcloud/lib/private/legacy/OC_App.php","line":205,"function":"bootApp","class":"OC\\AppFramework\\Bootstrap\\Coordinator","type":"->","args":["files_trashbin"]},{"file":"/data-ro/nextcloud/lib/private/legacy/OC_App.php","line":138,"function":"loadApp","class":"OC_App","type":"::","args":["files_trashbin"]},{"file":"/data-ro/nextcloud/apps/dav/lib/AppInfo/Application.php","line":124,"function":"loadApps","class":"OC_App","type":"::","args":[["dav"]]},{"file":"/data-ro/nextcloud/lib/private/AppFramework/Bootstrap/Coordinator.php","line":178,"function":"boot","class":"OCA\\DAV\\AppInfo\\Application","type":"->","args":[{"__class__":"OC\\AppFramework\\Bootstrap\\BootContext"}]},{"file":"/data-ro/nextcloud/lib/private/legacy/OC_App.php","line":205,"function":"bootApp","class":"OC\\AppFramework\\Bootstrap\\Coordinator","type":"->","args":["dav"]},{"file":"/data-ro/nextcloud/lib/private/legacy/OC_App.php","line":138,"function":"loadApp","class":"OC_App","type":"::","args":["dav"]},{"file":"/data-ro/nextcloud/lib/private/legacy/OC_Util.php","line":203,"function":"loadApps","class":"OC_App","type":"::","args":[["filesystem"]]},{"file":"/data-ro/nextcloud/lib/private/User/Session.php","line":554,"function":"setupFS","class":"OC_Util","type":"::","args":["*** sensitive parameter replaced ***"]},{"file":"/data-ro/nextcloud/lib/private/User/Session.php","line":415,"function":"prepareUserLogin","class":"OC\\User\\Session","type":"->","args":[true,"*** sensitive parameter replaced ***"]},{"file":"/data-ro/nextcloud/lib/private/User/Session.php","line":624,"function":"completeLogin","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/data-ro/nextcloud/lib/private/User/Session.php","line":367,"function":"loginWithPassword","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/data-ro/nextcloud/lib/private/Setup.php","line":440,"function":"login","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/data-ro/nextcloud/core/Command/Maintenance/Install.php","line":108,"function":"install","class":"OC\\Setup","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/data-ro/nextcloud/3rdparty/symfony/console/Command/Command.php","line":255,"function":"execute","class":"OC\\Core\\Command\\Maintenance\\Install","type":"->","args":[{"__class__":"Symfony\\Component\\Console\\Input\\ArgvInput"},{"__class__":"Symfony\\Component\\Console\\Output\\ConsoleOutput"}]},{"file":"/data-ro/nextcloud/3rdparty/symfony/console/Application.php","line":1009,"function":"run","class":"Symfony\\Component\\Console\\Command\\Command","type":"->","args":[{"__class__":"Symfony\\Component\\Console\\Input\\ArgvInput"},{"__class__":"Symfony\\Component\\Console\\Output\\ConsoleOutput"}]},{"file":"/data-ro/nextcloud/3rdparty/symfony/console/Application.php","line":273,"function":"doRunCommand","class":"Symfony\\Component\\Console\\Application","type":"->","args":[{"__class__":"OC\\Core\\Command\\Maintenance\\Install"},{"__class__":"Symfony\\Component\\Console\\Input\\ArgvInput"},{"__class__":"Symfony\\Component\\Console\\Output\\ConsoleOutput"}]},{"file":"/data-ro/nextcloud/3rdparty/symfony/console/Application.php","line":149,"function":"doRun","class":"Symfony\\Component\\Console\\Application","type":"->","args":[{"__class__":"Symfony\\Component\\Console\\Input\\ArgvInput"},{"__class__":"Symfony\\Component\\Console\\Output\\ConsoleOutput"}]},{"file":"/data-ro/nextcloud/lib/private/Console/Application.php","line":215,"function":"run","class":"Symfony\\Component\\Console\\Application","type":"->","args":[{"__class__":"Symfony\\Component\\Console\\Input\\ArgvInput"},{"__class__":"Symfony\\Component\\Console\\Output\\ConsoleOutput"}]},{"file":"/data-ro/nextcloud/console.php","line":100,"function":"run","class":"OC\\Console\\Application","type":"->","args":[]},{"file":"/data-ro/nextcloud/occ","line":11,"args":["/data-ro/nextcloud/console.php"],"function":"require_once"}],"File":"/data-ro/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","Line":115,"CustomMessage":"Could not boot files_trashbinCould not resolve trashManager! Class trashManager does not exist"},"userAgent":"--","version":"21.0.4.1"}
{"reqId":"hFGAKNvfQarjaxXVfGn3","level":3,"time":"2021-10-07T20:26:59+00:00","remoteAddr":"","user":"ncp","app":"no app in context","method":"","url":"--","message":{"Exception":"OCP\\AppFramework\\QueryException","Message":"Could not resolve OCA\\Files_Versions\\Versions\\IVersionManager! Class can not be instantiated","Code":0,"Trace":[{"file":"/data-ro/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":126,"function":"resolve","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["OCA\\Files_Versions\\Versions\\IVersionManager"]},{"file":"/data-ro/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":459,"function":"query","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["OCA\\Files_Versions\\Versions\\IVersionManager"]},{"file":"/data-ro/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":431,"function":"queryNoFallback","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->","args":["OCA\\Files_Versions\\Versions\\IVersionManager"]},{"file":"/data-ro/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":56,"function":"query","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->","args":["OCA\\Files_Versions\\Versions\\IVersionManager"]},{"file":"/data-ro/nextcloud/apps/files_versions/lib/AppInfo/Application.php","line":121,"function":"get","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["OCA\\Files_Versions\\Versions\\IVersionManager"]},{"file":"/data-ro/nextcloud/apps/files_versions/lib/AppInfo/Application.php","line":108,"function":"loadBackend","class":"OCA\\Files_Versions\\AppInfo\\Application","type":"->","args":[{"@attributes":{"for":"OCP\\Files\\Storage\\IStorage"},"@value":"OCA\\Files_Versions\\Versions\\LegacyVersionsBackend"},{"__class__":"OC\\AppFramework\\DependencyInjection\\DIContainer"},{"__class__":"OC\\AppFramework\\Logger"}]},{"file":"/data-ro/nextcloud/lib/private/AppFramework/Bootstrap/FunctionInjector.php","line":68,"function":"registerVersionBackends","class":"OCA\\Files_Versions\\AppInfo\\Application","type":"->","args":[{"__class__":"OC\\AppFramework\\DependencyInjection\\DIContainer"},{"__class__":"OC\\App\\AppManager"},{"__class__":"OC\\AppFramework\\Logger"}]},{"file":"/data-ro/nextcloud/lib/private/AppFramework/Bootstrap/BootContext.php","line":52,"function":"injectFn","class":"OC\\AppFramework\\Bootstrap\\FunctionInjector","type":"->","args":[{"__class__":"Closure"}]},{"file":"/data-ro/nextcloud/apps/files_versions/lib/AppInfo/Application.php","line":93,"function":"injectFn","class":"OC\\AppFramework\\Bootstrap\\BootContext","type":"->","args":[{"__class__":"Closure"}]},{"file":"/data-ro/nextcloud/lib/private/AppFramework/Bootstrap/Coordinator.php","line":178,"function":"boot","class":"OCA\\Files_Versions\\AppInfo\\Application","type":"->","args":[{"__class__":"OC\\AppFramework\\Bootstrap\\BootContext"}]},{"file":"/data-ro/nextcloud/lib/private/legacy/OC_App.php","line":205,"function":"bootApp","class":"OC\\AppFramework\\Bootstrap\\Coordinator","type":"->","args":["files_versions"]},{"file":"/data-ro/nextcloud/lib/private/legacy/OC_App.php","line":138,"function":"loadApp","class":"OC_App","type":"::","args":["files_versions"]},{"file":"/data-ro/nextcloud/apps/dav/lib/AppInfo/Application.php","line":124,"function":"loadApps","class":"OC_App","type":"::","args":[["dav"]]},{"file":"/data-ro/nextcloud/lib/private/AppFramework/Bootstrap/Coordinator.php","line":178,"function":"boot","class":"OCA\\DAV\\AppInfo\\Application","type":"->","args":[{"__class__":"OC\\AppFramework\\Bootstrap\\BootContext"}]},{"file":"/data-ro/nextcloud/lib/private/legacy/OC_App.php","line":205,"function":"bootApp","class":"OC\\AppFramework\\Bootstrap\\Coordinator","type":"->","args":["dav"]},{"file":"/data-ro/nextcloud/lib/private/legacy/OC_App.php","line":138,"function":"loadApp","class":"OC_App","type":"::","args":["dav"]},{"file":"/data-ro/nextcloud/lib/private/legacy/OC_Util.php","line":203,"function":"loadApps","class":"OC_App","type":"::","args":[["filesystem"]]},{"file":"/data-ro/nextcloud/lib/private/User/Session.php","line":554,"function":"setupFS","class":"OC_Util","type":"::","args":["*** sensitive parameter replaced ***"]},{"file":"/data-ro/nextcloud/lib/private/User/Session.php","line":415,"function":"prepareUserLogin","class":"OC\\User\\Session","type":"->","args":[true,"*** sensitive parameter replaced ***"]},{"file":"/data-ro/nextcloud/lib/private/User/Session.php","line":624,"function":"completeLogin","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/data-ro/nextcloud/lib/private/User/Session.php","line":367,"function":"loginWithPassword","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/data-ro/nextcloud/lib/private/Setup.php","line":440,"function":"login","class":"OC\\User\\Session","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/data-ro/nextcloud/core/Command/Maintenance/Install.php","line":108,"function":"install","class":"OC\\Setup","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/data-ro/nextcloud/3rdparty/symfony/console/Command/Command.php","line":255,"function":"execute","class":"OC\\Core\\Command\\Maintenance\\Install","type":"->","args":[{"__class__":"Symfony\\Component\\Console\\Input\\ArgvInput"},{"__class__":"Symfony\\Component\\Console\\Output\\ConsoleOutput"}]},{"file":"/data-ro/nextcloud/3rdparty/symfony/console/Application.php","line":1009,"function":"run","class":"Symfony\\Component\\Console\\Command\\Command","type":"->","args":[{"__class__":"Symfony\\Component\\Console\\Input\\ArgvInput"},{"__class__":"Symfony\\Component\\Console\\Output\\ConsoleOutput"}]},{"file":"/data-ro/nextcloud/3rdparty/symfony/console/Application.php","line":273,"function":"doRunCommand","class":"Symfony\\Component\\Console\\Application","type":"->","args":[{"__class__":"OC\\Core\\Command\\Maintenance\\Install"},{"__class__":"Symfony\\Component\\Console\\Input\\ArgvInput"},{"__class__":"Symfony\\Component\\Console\\Output\\ConsoleOutput"}]},{"file":"/data-ro/nextcloud/3rdparty/symfony/console/Application.php","line":149,"function":"doRun","class":"Symfony\\Component\\Console\\Application","type":"->","args":[{"__class__":"Symfony\\Component\\Console\\Input\\ArgvInput"},{"__class__":"Symfony\\Component\\Console\\Output\\ConsoleOutput"}]},{"file":"/data-ro/nextcloud/lib/private/Console/Application.php","line":215,"function":"run","class":"Symfony\\Component\\Console\\Application","type":"->","args":[{"__class__":"Symfony\\Component\\Console\\Input\\ArgvInput"},{"__class__":"Symfony\\Component\\Console\\Output\\ConsoleOutput"}]},{"file":"/data-ro/nextcloud/console.php","line":100,"function":"run","class":"OC\\Console\\Application","type":"->","args":[]},{"file":"/data-ro/nextcloud/occ","line":11,"args":["/data-ro/nextcloud/console.php"],"function":"require_once"}],"File":"/data-ro/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","Line":111,"CustomMessage":"Could not boot files_versionsCould not resolve OCA\\Files_Versions\\Versions\\IVersionManager! Class can not be instantiated"},"userAgent":"--","version":"21.0.4.1"}

```
</details>

**Comment by Fooughhy** [comment]
This is an issue I am getting as well. I am using docker, and it is much more cumbersome to change the "config.php" file because of that, so I'd like to use the UI. But it seems like it's encoding the string which is my domain name to some random string such as the one in OP (my local IP addresses seem to work fine).

**Comment by vpn** [comment]
Having this problem as well.

**Comment by nachoparker** [comment]
should be fixed in `v1.46.3`

thanks
